packages/@okta/vuepress-site/docs/reference/index.md

```diff
@@ -1,31 +1,25 @@
 ---
-component: Reference
-title: API Reference
-top_links:
-  - name: API Concepts
-    path: /docs/reference/api-overview/
-    icon: icons/icon--docs-apiconcepts.svg
-  - name: Error Codes
-    path: /docs/reference/error-codes/
-    icon: icons/icon--docs-errorcodes.svg
-  - name: Okta Expression Language
-    path: /docs/reference/okta-expression-language/
-    icon: icons/icon--docs-expressionlang.svg
-  - name: Postman Collections
-    path: /docs/reference/postman-collections/
-    icon: icons/icon--docs-postman.svg
+title: Reference overview
 ---
 
-::: slot left
+# Reference overview
+
+Details on parameters, requests, and responses for Okta's API endpoints.
+
+
+Okta APIs generally fall two categories: signing in your users, and managing
+resources in your Okta org.
+
 ## Sign in Your Users
```

Comment:
Sign in your users

(Sentence style capitalization in headers)

Reply:
done